docs/user_guide.md

```diff
@@ -597,3 +598,42 @@ Build history make it easy to see the contents of a container image, find the co
 In Harbor portal, enter your project, select the repository, click on the link of tag name you'd like to see its build history, the detail page will be opened. Then switch to `Build History` tab, you can see the build history information.
 
 ![build_ history](img/build_history.png)
+
+## Robot Account
+Robot accounts are accounts created by project admin that are intended for automated operations, 
+
+1, Robot account cannot login Harbor portal.
+2, Robot account can do docker pull/push only with token.
+
+### Add a robot account
+If you are a project admin, you can create a robot account by clicking "New Robot Account" in the **'Robot Accounts'** tab of a project, and enter a name, a description and permission.
+![add_robot_account](img/robotaccount/add_robot_account.png)
+
+![add_robot_account](img/robotaccount/add_robot_account_2.png)
+
+**NOTES:** The name will become robot$accountname that used to distinguish a robot account from harbor user.
+
+![copy_robot_account_token](img/robotaccount/copy_robot_account_token.png)
+As Harbor doesn't store your account token, please make sure to copy it in the pop up dialog after creating, otherwise, there is no way to get it from Harbor.

```

Comment:
Is this actually the case? Or do we just not show it to the user after it's generated? I'd think it'd *have* to be stored to verify the robot during `push`/`pull` operations...

Reply:
Per security concern, Harbor doesn't store the token of robot accounts. The only way to get it is from the pop-up dialog.

Reply:
Oh I see, it's a JWT token. Makes sense to me now!